Data/market-hours/market-hours-database.json

```diff
@@ -19515,7 +19515,7 @@
         },
         {
           "start": "08:30:00",
-          "end": "13:55:00",
+          "end": "16:00:00",
```

Comment:
> Sundays at 5:00 p.m. through Friday at 1:55 p.m. with daily trading halts from 4:00 p.m. to 5:00 p.m.

Think on friday it would end at `1:55 p.m` as in master?